suites/pacific/cephadm/tier-2_cephadm_conf_rgw_with_warn_state.yaml

```diff
@@ -0,0 +1,122 @@
+---
```

Comment:
change suite name to `tier-2-rgw-with-warn-state.`

Reply:
Thanks @vamahaja 
Updated